Mathlib/Algebra/DirectSum/Basic.lean

```diff
@@ -90,6 +90,10 @@ end AddCommGroup
 
 variable [∀ i, AddCommMonoid (β i)]
 
+@[ext] theorem ext {x y : DirectSum ι β} (w : ∀ i, x i = y i) :
+    x = y :=
```

Comment:
Does `@[ext]` generate the corresponding `ext_iff` lemma? If not, could you add it as well?

bors d+

Reply:
Yes, it generates it.